script/nnbd_plugins.sh

```diff
@@ -40,7 +40,7 @@ readonly NNBD_PLUGINS_LIST=(
 
 readonly NON_NNBD_PLUGINS_LIST=(
   "extension_google_sign_in_as_googleapis_auth"
-  "google_maps_flutter" # partially migrated
+  "google_maps_flutter_web" # Not yet migrated.
```

Comment:
😭

(Is this needed? I think `_web` packages are excluded by most of the CI scripts for now.)

Reply:
For now, yes:

> Because every version of google_maps_flutter_web from path depends on stream_transform ^1.2.0 and every version of google_maps_flutter_platform_interface from path depends on stream_transform ^2.0.0-nullsafety.0, google_maps_flutter_web from path is incompatible with google_maps_flutter_platform_interface from path.
> 
> So, because all_plugins depends on both google_maps_flutter_platform_interface from path and google_maps_flutter_web from path, version solving failed.

After this lands we can see about doing another PR that relaxes the version requirements in google_maps_flutter_web to turn it back on.